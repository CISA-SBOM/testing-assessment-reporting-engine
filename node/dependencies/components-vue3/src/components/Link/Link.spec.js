import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Link.vue'

describe('Link', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {})
    expect(wrapper.element).toMatchSnapshot()
  })
})
