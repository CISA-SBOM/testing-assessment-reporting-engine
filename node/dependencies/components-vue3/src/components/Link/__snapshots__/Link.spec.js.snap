// Vitest Snapshot v1

exports[`Link > should match its default snapshot 1`] = `
<a
  class=""
  data-id="sds-link"
>
  
  
</a>
`;
